src/scss/components/_footer.scss

```diff
@@ -6,157 +6,175 @@
 **/
 
 .footer {
-  @include media(medium) {
-    max-height: map-deep-get($footerSize, mini, size) + px;
-  }
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-  text-align: center;
-  width: 100%;
-
-  &--sticky {
-    bottom: 0;
-    justify-content: flex-end;
-    left: 0;
-    max-height: map-deep-get($footerSize, large, size) + px;
-    position: absolute;
-  }
+  background-color: get-color(wall);
+  height: auto;
 
-  &__company-info {
-    background-color: get-color(blackNeutral);
-    padding: {
-      bottom: 24px;
-      top: 24px;
-    };
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: 900;
```

Comment:
@eduardojmatos Vale usar o helper de font-weight? `get-weight`

Reply:
Yes. Scorpion is with this font-weight without helper...